tests/integrationv2/test_client_authentication.py

```diff
@@ -275,10 +275,13 @@ def test_tls_12_client_auth_downgrade(managed_process):
 
     # A s2n server built with OpenSSL1.0.2 and enabling client auth will downgrade the protocol to TLS1.2.
```

Comment:
I can't comment on the actual line, but I do have to ask-- why GnuTLS as the client? That's not really the most common choice of test client, so maybe add a comment explaining?

Not a blocker, I'm just curious.

Reply:
The client needs to support TLS 1.3 (so not s2n), and support TLS 1.2 so the handshake can proceed after the downgrade. It might be possible to configure this somehow with OpenSSL, but GnuTLS is configured this way by default. I added a comment.